modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableHeaderRow.java

```diff
@@ -291,13 +291,28 @@ public TableHeaderRow(final TableViewSkinBase skin) {
             dragHeader.setVisible(isReordering());
         }
     };
-    final void setReordering(boolean value) {
+
+    /**
+     * Inform this TableHeaderRow about a column being reordered.
+     * @param value true if a column is being reordered.
+     */
+    public final void setReordering(boolean value) {
         this.reordering.set(value);
     }
-    final boolean isReordering() {
+
+    /**
+     *
+     * @return true if a reorder is going on.
+     */
+    public final boolean isReordering() {
         return reordering.get();
     }
-    final BooleanProperty reorderingProperty() {
+
+    /**
+     *
+     * @return A BooleanProperty indicating whether a reorder is going on.
+     */
+    public final BooleanProperty reorderingProperty() {
```

Comment:
If anyone has overriden the TableHeaderRow, he may wants to be notified when a reorder is taking place. He also may wants to programmatically trigger a reorder. These methods should therefore be made protected.